### DISCUSSION TITLE
Add more options to FPS Limiter

### DISCUSSION THREAD

**Discussion by Mimonsi** [voxeltycoon/issues#893] [UI&UX]
I'd like to have more options for the FPS-Limiter than the current 30 and 60 Frames. I use a 240 hz monitor, Options for 120, 144, 240 and 300 FPS would be great.

**Comment by andrewpey** [comment]
Added `75`, `144`, and `240` Target FPS options, available in the latest [beta branch](https://voxeltycoon.xyz/devlog/beta-branch/) build.
